src/DotNetWorker/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -66,7 +68,8 @@ public static IFunctionsWorkerApplicationBuilder AddFunctionsWorkerCore(this ISe
 
         /// <summary>
         /// Adds the core set of services for the Azure Functions worker.
-        /// This call also adds the default set of binding converters and gRPC support.
+        /// This call adds the default set of binding converters and gRPC support.
+        /// This call also adds a default ObjectSerializer that treats property names as case insensitive.
```

Comment:
Maybe add this to the lists of things we do by default in `ConfigureFunctionsWorkerDefaults`? That's the API most people will interact with.